themes/default/content/docs/get-started/install/_index.md

```diff
@@ -27,12 +27,14 @@ This page contains detailed instructions for [installing Pulumi](#installing-pul
 
 macOS Sierra (10.12) or later is required.
 
-### Homebrew
+### Official Pulumi Homebrew Tap
 
-You can install Pulumi through the [Homebrew package manager](https://brew.sh/):
+You can install Pulumi through the [Homebrew package manager](https://brew.sh/) and using our official
+[Pulumi Homebrew Tap](https://github.com/pulumi/homebrew-tap/)
 
 ```bash
-$ brew install pulumi
+$ brew tap pulumi/tap
```

Comment:
Is this line needed?

I noticed another project I used recently didn't include this as a separate step, and it worked anyway:

https://nitric.io/docs/installation

Reply:
Good call - we don’t technically need it